index.in.html

```diff
@@ -53,7 +53,16 @@ <h3>R framework</h3>
 <!--include R-builds.html-->
 
 <p>
-The installer image (<tt>*.pkg</tt>) is packaged exactly the same way as the CRAN release of R (including the GUI) and it will update your R version (unless you use <tt>pkgutil</tt> - see instructions during installation). Alternatively you can use the tar-ball (<tt>*.tar.gz</tt>) which must be unpacked in the root (e.g. <tt>tar fvxz R*.tar.gz -C /</tt>), but doesn't contain the GUI (see below for a separate download).
+The installer image (<tt>*.pkg</tt>) is packaged exactly the same way as the CRAN release of R (including the GUI) and it will update your R version (unless you use <tt>pkgutil</tt> - see instructions during installation and/or the <i>"Multiple versions"</i> <a href="https://cran.r-project.org/doc/manuals/r-devel/R-admin.html#Multiple-versions">section of the R Installation and Administration manual</a>).
+
+<p>
+Alternatively you can use the tar-ball (<tt>*.tar.gz</tt>) which must be unpacked in the root:
+<blockquote>
+$ <tt>tar fvxz R*.tar.gz -C /</tt>
+</blockquote>
+<p>
+The tar-ball does not contain the GUI (see below for a separate download).
```

Comment:
Feels like this should also be prefixed with a `NOTE`

```suggestion
NOTE: The tar-ball does not contain the GUI (see below for a separate download).
```